validator/bin/reflector

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+ROOT=$(realpath $(dirname $0)/../..)
+
+if [[ $# < 4 ]]; then
+    echo Usage: $0 site_dir project_id device_id directive [directives...]
+    echo "  Directive is something like update/config:sites/udmi_site_model/devices/AHU-1/out/generated_config.json"
+    false
+fi
+
+site_dir=$(realpath $1)
+project_id=$2
+device_id=$3
+shift 3
+cd $ROOT
+
+validator/bin/build
+
+jarfile=validator/build/libs/validator-1.0-SNAPSHOT-all.jar
+mainclass=com.google.daq.mqtt.validator.Reflector
+
+cmd="java -cp $jarfile $mainclass -p $project_id -s $site_dir -d $device_id $*"
```

Comment:
Why not just invoke directly and "$*" on end?

Reply:
there was supposed to be another diagnostic "echo" in there... added!